answers.yaml.example

```diff
@@ -0,0 +1,7 @@
+dotfiles_repo: git@github.com:myuser/dotfiles.git
+fedora_packages:
+  my-fave-text-editor
+  my-alternate-web-browser
+fedora_packages_to_remove:
+  something-i-dislike
+  that-other-thing-i-dislike
```

Comment:
Nothing seems to read this. Is this intended to be read with say an include_vars task? Something that can optionally try to read the file, or even better have a config file somewhere that points to a git repo that has all the vars for a given box under SCM. Clone with one task, include_vars the content within in another task, sky's the limit from there?